packages/react/package.json

```diff
@@ -23,14 +23,14 @@
     "test": "jest --maxWorkers=1 --coverage"
   },
   "dependencies": {
-    "@popperjs/core": "^2.5.4",
+    "@popperjs/core": "^2.11.8",
     "classnames": "^2.2.6",
-    "focus-trap-react": "^10.2.3",
+    "focus-trap-react": "^10.3.0",
     "focusable": "^2.3.0",
     "keyname": "^0.1.0",
     "react-id-generator": "^3.0.1",
-    "react-popper": "^2.2.4",
-    "react-syntax-highlighter": "^15.5.0",
+    "react-popper": "^2.3.0",
+    "react-syntax-highlighter": "^15.6.1",
```

Comment:
There's quite a few dependency updates here, were they all necessary to support 18?

Reply:
I don't believe they were all necessary but I did update a few to see if it would fix a test but realized they weren't the  problem.

Reply:
Generally it's a good practice to try to avoid dependency and feature changes in the same PR since it might be hard to track down bugs later, unless they're necessary to address specific issues with a PR. My preference would be to revert and implement separately if we can.

Reply:
Updated here https://github.com/dequelabs/cauldron/pull/1731/commits/afd4f6ad92477b275b1628f375f4b5bf9ed61049